### PR TITLE
Document --skip_orca_root_stats

### PR DESCRIPTION
**Pull request by fifthposition**
Document analyzedb `--skip_orca_root_stats` option. Also removes `--skip_root_stats`.

### CODE REVIEW

**Comment by fifthposition** [comment]
cc @dyozie @zellerh (I don't seem to be able to request reviews)

File: gpdb-doc/dita/utility_guide/ref/analyzedb.xml [hunk 1]
```diff
@@ -168,15 +168,19 @@ public.lineitem -i l_shipdate, l_receiptdate </codeblock></pd>
             can be an integer between 1 and 10, inclusive. Default value is 5. </pd>
         </plentry>
         <plentry>
-          <pt>--skip_root_stats</pt>
+          <pt>--skip_orca_root_stats</pt>
           <pd>
-            <note>This option is deprecated and will be removed in a future release.</note>
+            <note>Do not use this option if GPORCA is enabled.</note>
           </pd>
           <pd>If the option is specified, a warning is issued stating that the option will be
```
Comment (zellerh) [inline gpdb-doc/dita/utility_guide/ref/analyzedb.xml:175]: > If the option is specified, a warning is issued stating that the option will be ignored.

Could you delete this, it applied to the old `--skip_root_stats` options.

**Review by zellerh — APPROVED** [review]
Thanks, this is nice and short.

File: gpdb-doc/dita/utility_guide/ref/analyzedb.xml [hunk 2]
```diff
@@ -168,15 +168,19 @@ public.lineitem -i l_shipdate, l_receiptdate </codeblock></pd>
...
             ignored.</pd>
-          <pd>GPORCA uses root partition statistics. For information about how statistics are
-            collected for partitioned tables, see <codeph><xref
-                href="../../ref_guide/sql_commands/ANALYZE.xml">ANALYZE</xref></codeph>.</pd>
+          <pd>Use this option if you find that <codeph>ANALYZE ROOTPARTITION</codeph> commands take
+            a very long time to complete.</pd>
+          <pd>
+            <note type="warning">After analyzedb is run with this option, subsequent analyzedb
```
Comment (dyozie) [inline gpdb-doc/dita/utility_guide/ref/analyzedb.xml:180]: add `codeph` style to the "analyzedb" entries here.

Also change:

analyzedb is run -> you run analyzedb

**Review by dyozie — APPROVED** [review]

File: gpdb-doc/dita/utility_guide/ref/analyzedb.xml (hunk 1 above)
Comment (fifthposition) [reply gpdb-doc/dita/utility_guide/ref/analyzedb.xml]: Done!

File: gpdb-doc/dita/utility_guide/ref/analyzedb.xml (hunk 2 above)
Comment (fifthposition) [reply gpdb-doc/dita/utility_guide/ref/analyzedb.xml]: Done!

**Pull request merged**